shuttle.go

```diff
@@ -392,7 +393,7 @@ func (cm *ContentManager) handleRpcTransferStatus(ctx context.Context, handle st
 		}
 
 		param.State = &filclient.ChannelState{
-			Status:  param.State.Status,
+			Status:  datatransfer.Failed,
```

Comment:
@alvin-reyes @jlogelin why this change? I used `state.status` because a failed state could be either `canceled` or `failed` - I wanted to retain the actual failed status.